app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -123,6 +126,12 @@ class SelectionInteractionViewModel private constructor(
     return when {
       isCurrentlySelected -> {
         selectedItems -= itemIndex
+        selectedItemsCount.set(selectedItems.size)
+        when (selectedItemsCount.get()) {
+          1 -> selectedItemText.set("You may select more choices")
```

Comment:
These cases were just for your understanding in real code we cannot use `1`, `2` or `3`.

Instead it will be 

`0` / default / else -> `Please select one or more choices.`

You will need to fetch `maxAllowableSelectionCount` from [here](https://github.com/oppia/oppia-android/blob/dbd66d5c183f1d965b59ce8b62b80b332f3cc9a7/domain/src/main/assets/test_exp_id_2.json#L1155)

which will be used to define other two strings.

your code will be something like this:
```
if(selectedItemsCount.get() == 0){
    selectedItemText.set("Please select one or more choices.")
}
else if (selectedItemsCount.get() < maxAllowableSelectionCount){
    selectedItemText.set("You may select more choices")
}
else if(selectedItemsCount.get() == maxAllowableSelectionCount){
    selectedItemText.set("No more than $maxAllowableSelectionCount choices may be selected.")
    // After this you will need to make sure that the selection is disabled so that user cannot select anymore items.
}

```

This code has not been tested.

Reply:
okay @rt4914